thoth/storages/graph/models.py

```diff
@@ -548,6 +548,7 @@ class AdviserRun(Base, BaseExtension):
     count = Column(Integer, nullable=True)
     limit = Column(Integer, nullable=True)
     origin = Column(Text, nullable=True)
+    source_type = Column(Text, nullable=True)
```

Comment:
Enum? The database will not grow that significantly as storing enum will be more efficient in comparison to a string. Also, the queries will be more efficient - no need to do a string comparison.

Reply:
@fridex seems to give an error on doing an `alembic upgrade head`, complete error log - https://bit.ly/2YJjm1T
The enum seems to be defined like all the other enums, just nullable set to `True`

Reply:
https://github.com/thoth-station/storages/blob/master/thoth/storages/graph/enums.py
Should I include the enum here again? Instead of `from thoth.common import ThothAdviserIntegrationEnum`?

Reply:
It has to be a database enum, something like this one:

https://github.com/thoth-station/storages/blob/361b64c30434ba0d6a6f6a182ee2ae66cb851f36/thoth/storages/graph/models.py#L45-L48

Reply:
> It has to be a database enum, something like this one:
> 
> https://github.com/thoth-station/storages/blob/361b64c30434ba0d6a6f6a182ee2ae66cb851f36/thoth/storages/graph/models.py#L45-L48

Thanks for explaining.